rsg

```diff
@@ -36,14 +37,30 @@ def main():
         shell_type = argv[3].upper()
     else:
         shell_type = ''
+    number_of_shells = 0
     for shell in open(shells):
+        number_of_shells = number_of_shells + 1
         desc, cmd = shell.split('|', 1)
         cmd = cmd.replace("[IPADDR]", ipaddr)
         cmd = cmd.replace("[PORT]", port)
         if shell_type in desc:
             print(GREEN(desc))
             print(cmd)
 
+    c = input('Copy your payload by pressing the number or enter to exit: ')
```

Comment:
Should we exit here or continue to see if they want to listen as well as copy?

Reply:
Hi

@roguepullrequest 

What do u think abput those options:
 
1. listen only
2. copy & listen
3. exit

If u think those are good I can add them.

Reply:
Hi
@roguepullrequest 

I added the option and fixed a bug for the last shell